scripts/install2

```diff
@@ -14,12 +14,16 @@ BOARD_HARDWARE=$(cat /proc/cpuinfo  | grep Hardware | awk '{print $3}' | head -n
 # Possibly use /sys/firmware/devicetree/base/model
 if [[ "$BOARD_HARDWARE" == 'Allwinner' || "$BOARD_HARDWARE" == 'Generic' ]]; then    
     BOARD_FAMILY="Orange Pi"
-    BOARD_NAME="Generic"
+    source /etc/armbian-image-release
+    BOARD_NAME=`echo $BOARD_NAME | sed 's/Orange Pi //g'`
```

Comment:
Isn't `BOARD_NAME` always "Generic" here?

Reply:
it gets sets with this line 
source /etc/armbian-image-release

Reply:
`/etc/armbian-image-release` is a file that comes with the Armbian image, and we are relying on it to set our `BOARD_NAME` variable?

Reply:
Correct
From my research its the ONLY way to identify the board 100% is by the image that is flashed. Each image includes this file

its content is as follows (for an zero)

root@orangepizero:~# cat /etc/armbian-image-release 
# PLEASE DO NOT EDIT THIS FILE
BOARD=orangepizero
BOARD_NAME="Orange Pi Zero"
VERSION=5.32.170920
LINUXFAMILY=sun8i
BRANCH=dev
ARCH=arm
IMAGE_TYPE=nightly
BOARD_TYPE=conf
INITRD_ARCH=arm
KERNEL_IMAGE_TYPE=zImage

Reply:
I think it's better to grep for the "BOARD_NAME" string value and set the local `BOARD_NAME` variable to the "Zero" than to source the entire file.

Reply:
It seems allot more cleaner/simpler then

``` BOARD_NAME=$(cat /etc/armbian-image-release | grep BOARD_NAME | awk -F \= '{print $2}' | tr -d \")```

but i can go either way...

Reply:
Don't need the `cat`, just `grep`. But this way it's more readable and we won't accidentally run unintended things in `/etc/armbian-image-release`.